deployment/build_and_release/modules/release/variables.tf

```diff
@@ -42,3 +42,30 @@ variable "cloudbuild_trigger_branch" {
   type        = string
   default     = ""
 }
+
+variable "build_substitutions" {
+  type = object({
```

Comment:
What's the motivation to put all of these in an object? I like the encapsulation[1], but I think this loses out on the ability to add default values in the module, and it's harder to specify some of these in the base `terragrunt.hcl`. e.g. `checkpoint_cache` and `entries_dir` are probably never going to change and could be defaulted here, or specified in the root `terragrunt.hcl`.

[1] Though the encapsulation isn't perfect, e.g. `var.env` is used in some of the steps.

Reply:
Ok, I don't mind, outdented them.